src/test/java/Homework21.java

```diff
@@ -0,0 +1,58 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework21 extends BaseTest {
+
+    public static String playlistName = "Test Playlist";
+
+    @Test
+    public static void renamePlaylist() throws InterruptedException {
```

Comment:
Please, delete `throws InterruptedException`

**Thread. sleep is bad!** It blocks the current thread and renders it unusable for further work.
Using Thread. sleep() frequently in an automation framework is not a good practice.

Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.

![image](https://user-images.githubusercontent.com/104761482/212639101-b5c8189c-4f85-4113-843a-61b5c3b9b8b7.png)

Reply:
Thank you for the feedback! It's much appreciated! I removed Thread.sleep and added explicit waits. I pushed my changes. Thanks again!